src/python/pants/backend/python/tasks2/pytest_run.py

```diff
@@ -235,7 +252,18 @@ def add_realpath(path):
         realpaths.add(realpath)
 
     cp = configparser.SafeConfigParser()
-    cp.readfp(StringIO(self.DEFAULT_COVERAGE_CONFIG))
+    coverage_config_file = self.get_options().coverage_config_file
+    if coverage_config_file is not None:
+      try:
+        with io.open(coverage_config_file, 'rt') as fp:
+          cp.readfp(fp)
+      except OSError as e:
+        # python cannot read this file, might be missing, or permissions are wrong
```

Comment:
Nit: Sentences should begin with a capital letter (unless the word is the name of a variable or something else case-sensitive) and end with a period.